lms/djangoapps/ccx/tasks.py

```diff
@@ -27,6 +28,7 @@ def course_published_handler(sender, course_key, **kwargs):  # pylint: disable=u
 
 
 @CELERY_APP.task
```

Comment:
Author's note: Should this be changed to `shared_task` or something? I'm not clear on how that aspect of Celery works.